File: src/main/java/com/example/dto/account/AccountFullDto.java
package com.example.dto.account;

import com.example.dto.card.CardDto;
import com.example.dto.image.ImageDto;
import com.example.dto.role.RoleDto;

import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.AllArgsConstructor;
import java.util.List;
import java.util.UUID;

@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class AccountFullDto {
    private UUID id;
    private String nickname;
    private String login;
    private ImageDto image;
    private List<CardDto> cards;
    private List<RoleDto> roles;
}

File: src/main/java/com/example/dto/account/AccountFullDto.kt
package com.example.dto.account

import com.example.dto.card.CardDto
import com.example.dto.image.ImageDto
import com.example.dto.role.RoleDto
import java.util.UUID

data class AccountFullDto(
    val uuid: UUID,
    var nickname: String?,
    val login: String,
    val image: ImageDto?,
    val cards: List<CardDto>?,
    val roles: List<RoleDto>?,
)
